src/v2/Components/NavBar/Menus/DropDownMenu/DropDownMenu.tsx

```diff
@@ -17,8 +17,12 @@ const LinkMenuItem = styled(MenuItem).attrs({
 
 LinkMenuItem.displayName = "LinkMenuItem"
 
-const ViewAllMenuItem = styled(LinkMenuItem).attrs({
+const ViewAllMenuItem = styled(MenuItem).attrs({
```

Comment:
Curious what's your thinking about not inheriting from `LinkMenuItem`. Is it to avoid too much inheritance?

Reply:
It was because the typings were somehow getting lost so `href` wasn't being picked up. I didn't want to dig too deep into why because I can see that turning into a *thing*.